events-amd.js

```diff
@@ -34,9 +34,20 @@ define('event', [], function() {
 	}
 
 	return {
-		addEvent : function(el, eventName, command, named) {
+		addEvent : function(el, eventName, command, namedOrConfigs) {
+			var named;
+			if (typeof namedOrConfigs === "string") {
+				named = namedOrConfigs;
+			} else if (typeof namedOrConfigs === "object") {
+				named = namedOrConfigs.named;
+			}
+
 			if (el.addEventListener) {
-				el.addEventListener(eventName, command);
+				if (namedOrConfigs && namedOrConfigs.passive){
```

Comment:
Instead of checking if the `nameOrConfig` has the passive atribute, what do you think of checking if the `nameOrConfig` is an `Object`? That way it's works for `capture` or `once` too.

@erikatakahara

Reply:
Suggestion: the `passive` event option does not works on IE11, it's throws an error instead. This one works great on our currently problem on mobile, but in the future we'll have to check if the browser supports it before calling this function. What do you think of making this check now? Or we can do it when we refactor the `doc-amd` too.

@erikatakahara